packages/code-du-travail-frontend/src/outils/DureePreavisDemission/steps/situation.js

```diff
@@ -168,3 +82,14 @@ export function recapSituation(criteria) {
       null
     );
 }
+
+export function getSituationsFor(obj) {
+  return data.filter(situation =>
+    Object.entries(obj).every(([key, value]) => situation[key] === value)
+  );
+}
+
+export const isNotYetProcessed = idcc => {
```

Comment:
same here this function could be more generic and move to utils (and probably used in IndemnitePrecarité)

Reply:
OK

Reply:
Done